Views/Shared/_Layout.cshtml

```diff
@@ -14,32 +14,43 @@
     <link href="~/css/Nunito.css" rel="stylesheet preload" as="style">
     <link href="~/lib/fortawesome/fontawesome-free/css/all.min.css" rel="stylesheet">
 
-    <!-- Custom styles for this template-->
+    <!-- SB ADMIN 2-->
     <link href="~/css/sb-admin-2.min.css" rel="stylesheet">
 
+    <!-- UI Template -->
+    <link href="~/css/styles.css" rel="stylesheet">
+
+    <link href="~/lib/daterangepicker/daterangepicker.css" rel="stylesheet">
     <link href="~/img/favicon.ico" rel="icon" type="image/x-icon">
 
+    <script src="https://unpkg.com/feather-icons"></script>
+    <meta name="theme-color" content="#4e73df">
     @RenderSection("Styles", false)
 </head>
 
 <body id="page-top">
 
+    <!--<div id="loader" style="height:100vh">
+        <iframe src="~/img/venjix.svg" style="height:100vh;width:100%">
+        </iframe>
+    </div>-->

```

Comment:
If this is unwanted, please remove it

Reply:
It's not, I'm just playing with loader or something but it's unnecessarily slow.